smriprep/workflows/anatomical.py

```diff
@@ -29,9 +29,20 @@
 
 
 def init_anat_preproc_wf(
-        bids_root, freesurfer, hires, longitudinal, omp_nthreads, output_dir,
-        output_spaces, num_t1w, reportlets_dir, skull_strip_template,
-        debug=False, name='anat_preproc_wf', skull_strip_fixed_seed=False):
+        bids_root,
+        freesurfer,
+        hires,
+        longitudinal,
+        num_t1w,
+        omp_nthreads,
+        output_dir,
+        reportlets_dir,
+        skull_strip_template,
+        spaces,
+        debug=False,
+        name='anat_preproc_wf',
+        skull_strip_fixed_seed=False
```

Comment:
Just a note that we've apparently abandoned our usual rule of no default arguments (except `name`) in workflow building functions.

We might want to think about either re-enforcing it or making an explicit statement in a style guide of when default arguments should be used.

Reply:
Sounds reasonable